app/src/main/java/com/door43/translationstudio/spannables/USFMVerseSpan.java

```diff
@@ -147,4 +151,37 @@ public static USFMVerseSpan parseVerse(String usfm) {
             return new int[]{startVerse, endVerse};
         }
     }
+
+    /**
+     * fix verse markers missing the terminating space after verse number.
+     * @param in
+     * @return
+     */
+    public static CharSequence fixIncompleteVerseMarkers(CharSequence in) {
```

Comment:
note: I'm not certain we want to do this, but I don't think there's any need to change it for now.